### PR TITLE
chore(odyssey-react): Adding resize attribute to TextArea

### PR DESCRIPTION
**Pull request by swapnil-nagar**
### Description

Adding the resize attribute to TextArea. 

[OKTA-464388](https://oktainc.atlassian.net/browse/OKTA-464388)

### CODE REVIEW

File: packages/odyssey-react/src/components/TextArea/TextArea.module.scss [hunk 1]
```diff
@@ -78,3 +78,19 @@
...
+.horizontalResize {
+  font-size: horizontal;
+}
+
+.verticalResize {
+  font-size: vertical;
+}
+
+.bothResize {
+  font-size: both;
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/TextArea/TextArea.module.scss:95]:
```suggestion
  resize: both;
```

same goes for the other utility classes above.

**Review by jeffbelser-okta — CHANGES_REQUESTED** [review]
strong work 💪 you'll want to update your commit title as well to match our convention. See our contribution guidelines for more detail there

Also please add a new storybook example to show a text area with `resize="none"` enabled so we can get VRT coverage

File: packages/odyssey-react/src/components/TextArea/TextArea.test.tsx [hunk 2]
```diff
@@ -109,5 +109,11 @@ describe("TextArea", () => {
     expect(ref).toHaveBeenLastCalledWith(screen.getByRole(textBox));
   });
 
+  it("check if resize is populates a correct class name", () => {
```
[the remark this answers is not in the document]
Comment (swapnil-nagar) [reply packages/odyssey-react/src/components/TextArea/TextArea.test.tsx]: Done

File: packages/odyssey-react/src/components/TextArea/TextArea.test.tsx [hunk 3]
```diff
@@ -109,5 +109,11 @@ describe("TextArea", () => {
     expect(ref).toHaveBeenLastCalledWith(screen.getByRole(textBox));
   });
 
+  it("assigns resize prop correctly", () => {
+    render(<TextArea resize="none" label={label}/>);
+    const container = screen.getByRole(textBox, { name: label });
+    expect(container).toHaveClass('noneResize');
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/TextArea/TextArea.test.tsx:115]: a slightly better test would involve the `toHaveStyle` assertion: https://github.com/testing-library/jest-dom#tohavestyle

**Comment by jeffbelser-okta** [comment]
rather than titling your commit `chore: adding resize attribute to textarea` I would suggest something like this:

`feat(odyssey-react): add resize prop to TextArea`

and then a second commit like

`docs(odyssey-storybook): add new resize story for TextArea`

**Review by jeffbelser-okta — APPROVED** [review]
This looks good to merge once we have green CI!

File: packages/odyssey-react/src/components/TextArea/TextArea.test.tsx (hunk 3 above)
Comment (swapnil-nagar) [reply packages/odyssey-react/src/components/TextArea/TextArea.test.tsx]: CSS never complied in test suite and current test should be ok.

**Pull request merged**